src/components/modern/LargeFiatTokenValueView.js

```diff
@@ -55,7 +55,7 @@ function LargeFiatTokenValueView({ value, assetAddress, chain, symbol, style }:
   const fiatCurrency = useFiatCurrency();
   const colors = useThemeColors();
   const rates = useChainRates(chain);
-  if (!value || !assetAddress) return null;
+  if (!value || value.isZero() || !assetAddress) return null;
```

Comment:
This check should not be done in LargeFiatTokenValue, since there might be legit cases when you would want to show "0 ETH". Please move it to calling component.

Reply:
Maciej I thought the same, but zero value transactions are data transactions so it kinda makes sense to have it at this level, but not saying it doesn't make sense to move it level up.

Reply:
This control is general purpose, not limited to tx value, so I would lean to moving the check up to the call site.